index.html

```diff
@@ -5,310 +5,243 @@
-                        <li class="lang2">javaScript</li>
-                    </ul>
-                    <button class="btn openPopup" id="secondBtn" onclick="changeWindowPopup(2)" type="submit">See
-                        Project</button>
-                </div>
-                <img class="work-pic-1" src="./images/desktop-images/img2.png" alt="professional art printing image" />
+        </div>
+        <div class="card reverse">
+            <div class="portfolio-img pic2">
+                <img class="pic" src="./pics/works img/work4.svg" alt="card1" />
             </div>
-            <div class="container">
-                <img class="work-pic" src="./images/Portoflio_Card/work2.png" alt="facebook 360 image" />
-                <img class="work-pic" src="./images/desktop-images/img3.png" alt="landing view, desktop-image" />
-                <div class="work-description">
-                    <h4>TONIC</h4>
-                    <ul class="description-element">
-                        <li>CANOPY</li>
-                        <li class="element">
-                            <img class="bullet" src="./images/bullet.png" alt="bullet" />
-                        </li>
-                        <li class="element">Back End Dev</li>
-                        <li class="element">
-                            <img class="bullet" src="./images/bullet.png" alt="bullet" />
-                        </li>
-                        <li class="element">2015</li>
-                    </ul>
-                    <p class="description-text descriptionTxt">
-                        A daily selection of privately personalized reads; no accounts or
-                        sign-ups required.
-                    </p>
-                    <ul class="languages">
-                        <li class="lang">html</li>                     
-                        <li class="lang1">css</li>
-                        <li class="lang2">javaScript</li>
-                    </ul>
-                    <button class="btn openPopup" id="thirdBtn" type="submit">See Project</button>
+            <div class="works-details">
+                <h1 class="comp-title">Multi-Post
+                    Stories</h1>
+                <ul class="work-title">
+                    <li class="client">FACEBOOK</li>
+                    <li><img src="pics/works img/Counter.svg" alt="dot" /></li>
+                    <li class="dim">Back End Dev</li>
+                    <li><img src="pics/works img/Counter.svg" alt="dot" /></li>
+                    <li class="dim">2015</li>
+                </ul>
+                <div class="project-desc">
+                    <p>Experimental content creation feature that allows users to add to an existing story over the
+                        course of a day without spamming their friends.</p>
                 </div>
+                <ul class="tags">
+                    <li class="tag">html</li>
+                    <li class="tag">Ruby on rails</li>
```

Comment:
- [ ] Please apply the same CSS rules to this `li` item so that it looks like the rest of the technologies.  ⛑️ 
![tech](https://user-images.githubusercontent.com/99656687/221349895-37dbf8f7-987d-4c1f-be6f-cd21985f1d74.png)